Bag of documentation fixes; fix more sphinx warnings

zou3519 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #27927 Fix Sphinx warning about '_images' not existing
* **#27850 Bag of documentation fixes; fix more sphinx warnings**

Many of these are real problems in the documentation (i.e., link or
bullet point doesn't display correctly).

Test Plan:
- built and viewed the documentation for each change locally.

Differential Revision: [D17908123](https://our.internmc.facebook.com/intern/diff/D17908123)

zou3519 committed:
Bag of documentation fixes; fix more sphinx warnings

Many of these are real problems in the documentation (i.e., link or
bullet point doesn't display correctly).

Test Plan:
- built and viewed the documentation for each change locally.

zou3519 commented on docs/source/quantization.rst:
```diff
@@ -186,7 +188,7 @@ Layers for the quantization-aware training
 
 * :class:`~torch.quantization.MinMaxObserver` — Derives the quantization parameters from the running minimum and maximum of the observed tensor inputs (per tensor variant)
-* :class:`~torch.quantization.MovingAverageObserver` — Derives the quantization parameters from the running averages of the minimums and maximums of the observed tensor inputs (per tensor variant)
+* ``MovingAverageObserver`` — Derives the quantization parameters from the running averages of the minimums and maximums of the observed tensor inputs (per tensor variant)
```
This doesn't exist in the documentation. I've filed an issue at https://github.com/pytorch/pytorch/issues/27849. There is a comment about it in a section below; I couldn't figure out how to insert a comment here without breaking up the bullet pointing.

zou3519 committed:
Update on "Bag of documentation fixes; fix more sphinx warnings"


Many of these are real problems in the documentation (i.e., link or
bullet point doesn't display correctly).

Test Plan:
- built and viewed the documentation for each change locally.

Differential Revision: [D17908123](https://our.internmc.facebook.com/intern/diff/D17908123)

zou3519 committed:
Update on "Bag of documentation fixes; fix more sphinx warnings"


Many of these are real problems in the documentation (i.e., link or
bullet point doesn't display correctly).

Test Plan:
- built and viewed the documentation for each change locally.

Differential Revision: [D17908123](https://our.internmc.facebook.com/intern/diff/D17908123)

zafartahirov commented on docs/source/quantization.rst:
```diff
@@ -186,7 +188,7 @@ Layers for the quantization-aware training
 
 * :class:`~torch.quantization.MinMaxObserver` — Derives the quantization parameters from the running minimum and maximum of the observed tensor inputs (per tensor variant)
-* :class:`~torch.quantization.MovingAverageObserver` — Derives the quantization parameters from the running averages of the minimums and maximums of the observed tensor inputs (per tensor variant)
+* ``MovingAverageObserver`` — Derives the quantization parameters from the running averages of the minimums and maximums of the observed tensor inputs (per tensor variant)
```
I think it will work if you change it to ```:class:`~torch.quantization.MovingAverageMinMaxObserver` ```

zou3519 replied:
That works, thank you for the pointer

zou3519 committed:
Update on "Bag of documentation fixes; fix more sphinx warnings"


Many of these are real problems in the documentation (i.e., link or
bullet point doesn't display correctly).

Test Plan:
- built and viewed the documentation for each change locally.

Differential Revision: [D17908123](https://our.internmc.facebook.com/intern/diff/D17908123)

zou3519 committed:
Update on "Bag of documentation fixes; fix more sphinx warnings"


Many of these are real problems in the documentation (i.e., link or
bullet point doesn't display correctly).

Test Plan:
- built and viewed the documentation for each change locally.

Differential Revision: [D17908123](https://our.internmc.facebook.com/intern/diff/D17908123)

zou3519 committed:
Update on "Bag of documentation fixes; fix more sphinx warnings"


Many of these are real problems in the documentation (i.e., link or
bullet point doesn't display correctly).

Test Plan:
- built and viewed the documentation for each change locally.

Differential Revision: [D17908123](https://our.internmc.facebook.com/intern/diff/D17908123)

zou3519 committed:
Update on "Bag of documentation fixes; fix more sphinx warnings"


Many of these are real problems in the documentation (i.e., link or
bullet point doesn't display correctly).

Test Plan:
- built and viewed the documentation for each change locally.

Differential Revision: [D17908123](https://our.internmc.facebook.com/intern/diff/D17908123)

[ghstack-poisoned]
